Engine/Plugins/Marketplace/RuntimeMeshImportExport/Source/RuntimeMeshImportExport/Private/AssimpCustom.cpp

```diff
@@ -412,6 +413,14 @@ void FAssimpNode::CreateAssimpMeshesFromMeshData(FAssimpScene& scene, const FRun
 						material->AddProperty(&diffuseTextureRelativePath, AI_MATKEY_TEXTURE(aiTextureType_DIFFUSE, 0));
 					}
 				}
+				// Set the metalness texture of the material
+				{
+					if (!section.materialToExport.metalnessTextureRelativePath.IsEmpty())
```

Comment:
Is there a reason for this metalness texture? Does it do anything?

Reply:
I added this first before the GLTF MetallicRougnessTexture, if we ever switch export format then we have a working metallic export. We should probably either remove this or add the roughness texture too.

Reply:
I would remove it I think. It's not something that will be difficult to re-implement later.